WalletWasabi.Tests/UnitTests/Transactions/TransactionProcessorTests.cs

```diff
@@ -244,12 +245,14 @@ public async Task HandlesRbfAsync()
 			tx = CreateSpendingTransaction(createdCoin, transactionProcessor.NewKey("E").P2wpkhScript);
 			tx.Transaction.Outputs[0].Value = Money.Coins(0.9m);
 			relevant = transactionProcessor.Process(tx);
-			Assert.Equal(1, replaceTransactionReceivedCalled);
 
 			Assert.True(relevant);
+			Assert.Equal(1, replaceTransactionReceivedCalled);
 			var finalCoin = Assert.Single(transactionProcessor.Coins);
 			Assert.True(finalCoin.IsReplaceable);
 			Assert.Equal("E", finalCoin.HdPubKey.Label);
+
+			Assert.False(transactionProcessor.Coins.AsAllCoinsView().Contains(unconfirmedCoin1));
```

Comment:
VS suggested to use Assert.DoesNotContain
```suggestion
			Assert.DoesNotContain(unconfirmedCoin1, transactionProcessor.Coins.AsAllCoinsView());
```